azure/template.json

```diff
@@ -107,14 +107,23 @@
         "sharedEnvVirtualNetworkName": {
             "type": "string"
         },
-        "subnetObject": {
+        "msgHandlerSubnetObject": {
```

Comment:
Having the full messageHandlerSubnetObject to be the parameter name similar to the rest would be best I reckon (eg: can ctrl+f for messageHandler)